tools/nx-plugins/npm-package/executors/compile/rollup.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = (config) => {
+	config.output.sourcemap = true;
+	config.output.preserveModules = true;
+	return config;
+};
```

Comment:
again, it's the fact it's actually going to be imported from a cjs file in node. it's a bit confusing i agree!

Reply:
https://nx.dev/packages/rollup/executors/rollup#rollupconfig

Reply:
Using JSDoc comments might be helpful?

```js

/** @typedef {import('rollup').RollupOptions} Config */

/**
 * @type {(config: Config) => Config}
 * @see https://nx.dev/packages/rollup/executors/rollup#rollupconfig
 */
export default (config) => {
 /* do your thing */
}
```

Reply:
we don't actually install rollup, it's a dep of the nrwl rollup plugin. therefore pnpm can't find it. the only way this would work would be to install rollup so we can get the types, but then we need to keep it in sync with the version that the plugin uses.

given we're only setting a few props and invalid config will be ignored, i think it's ok?